includes/class-ur-form-handler.php

```diff
@@ -583,6 +583,51 @@ public static function process_login() {
 	 */
 	public static function process_lost_password() {
 		if ( isset( $_POST['ur_reset_password'] ) && isset( $_POST['user_login'] ) && isset( $_POST['_wpnonce'] ) && wp_verify_nonce( sanitize_key( $_POST['_wpnonce'] ), 'lost_password' ) ) {
+
+			$hcaptca_response    = isset( $_POST['h-captcha-response'] ) ? sanitize_text_field( wp_unslash( $_POST['h-captcha-response'] ) ) : '';
```

Comment:
@bhattaganesh I think the codes below these lines are repeated in other places also. Please make a common function to handle this and implement it in another place as well.